README.md

```diff
@@ -403,23 +403,12 @@ metadata:
     app: notification-manager
     type: default
 spec:
-  conversation:
-    appkey: 
-      key: appkey
-      name: default-dingtalk-secret
-    appsecret:
-      key: appsecret
-      name: default-dingtalk-secret
-    chatid: chat894f9f4d634eb283933af6c7102977b2
-  chatbot:
-    webhook:
-      key: webhook
-      name: test-user-dingtalk-secret
-    keywords: 
-    - kubesphere
-    secret:
-      key: secret
-      name: test-user-dingtalk-secret
+  appkey: 
```

Comment:
I think we still need a `conversation` layer here in dingtalk config to be clear that this config is for conversation only